packages/common/src/utils/scheduler.ts

```diff
@@ -11,3 +11,30 @@ export function getHumanReadableCronExpression(cronExpression: string) {
 
     return valueWithUTC[0].toLowerCase() + valueWithUTC.slice(1);
 }
+
+export function isAllowedCronExpression(cronExpression: string): boolean {
+    /** This function will return False if:
+     * - the cronExpression is not valid (not 5 parts separated by spaces)
+     * - the cronExpression frequency is less than 1 hour
+     */
+    const cronParts = cronExpression.trim().split(' ');
+    if (cronParts.length !== 5) {
+        // Invalid cron expression
+        return false;
+    }
```

Comment:
Nitpick: this could be replaced by using `isInvalidCronExpression` although it is currently in `packages/frontend/src/utils/fieldValidators.ts` it would need to be moved to `common`

Reply:
I've been looking at that method, but the type and arguments are different, `isInvalidCronExpression` returns an `error` string, while my method returns a boolean. 
The only thing those 2 methods have in common is the `split().length !== 5` , which in my method is even optional.

What I can do is use the method from `common` inside the validator, let me try.

Reply:
updated